src/Core/Authorization/AuthorizationResolver.cs

```diff
@@ -525,13 +616,19 @@ private static string GetPolicyWithClaimValues(string policy, Dictionary<string,
     /// <param name="claimsInRequestContext">Dictionary populated with all the user claims.</param>
     /// <returns>The claim value for the given claimTypeMatch.</returns>
     /// <exception cref="DataApiBuilderException"> Throws exception when the user does not possess the given claim.</exception>
-    private static string GetClaimValueFromClaim(Match claimTypeMatch, Dictionary<string, Claim> claimsInRequestContext)
+    private static string GetClaimValueFromClaim(Match claimTypeMatch, Dictionary<string, List<Claim>> claimsInRequestContext)
     {
         // Gets <claimType> from @claims.<claimType>
         string claimType = claimTypeMatch.Value.ToString().Substring(CLAIM_PREFIX.Length);
-        if (claimsInRequestContext.TryGetValue(claimType, out Claim? claim))
+        if (claimsInRequestContext.TryGetValue(claimType, out List<Claim>? claims))
         {
-            return GetClaimValue(claim);
+            // Database policies do not support operators like "in" or "contains".
+            // Return the first value in the list of claims.
+            // This is not a breaking change since historically,
+            // if the user had >1 role AND wrote a db policy to include the token '@claims.role',
+            // DAB would fail the request. Now, the request won't fail, but the value
+            // resolved is the first claim encountered (when there are multiple claim instances).
+            return GetClaimValue(claims.First());
```

Comment:
nit: Add a `null` check for `claims` before using `.First()` method

Reply:
thanks for calling this out. While the list of claims won't be null/empty due because the code populating the list won't add the list/claim type to the dictionary if one doesn't exist, still good to have defensive code. Adding the condition:
```csharp
&& claims is not null && claims.Count > 0
```
so that if that condition is not met an exception is thrown.